match/match.py

```diff
@@ -278,6 +279,16 @@ async def _add_match(self, ctx, match_day, match_date, home, away, *args):
         result['away'] = away
         return result
 
+    async def _set_match_on_stream(self, ctx, match_day, team, stream_details):
+        matches = await self._matches(ctx)
+        for match in matches:
+            if match['matchDay'] == match_day and (one_team == match['home'] or one_team == match['away']):
```

Comment:
This loop functionality looks the same as in the get_match_from_day_team method. Is there a reason not to just call that method to get the match and edit it here if found? That way the functionality can be guaranteed to be consistent if it needs to change in the future.